Makefile

```diff
@@ -14,16 +14,15 @@ endif
 BASE_IMAGE_NAME ?= kuma_base
 KUMA_IMAGE_NAME ?= kuma
 KUMASCRIPT_IMAGE_NAME ?= kumascript
-REGISTRY ?= quay.io/
```

Comment:
This will make it complex to make a image with `mdnwebdocs` in local build. I think it can be handeled something ike following and pass the `PIPELINE_NAME` env file from the `Jenkinsfile`
```make
ifeq ($(PIPELINE_NAME), mozilla_it)
REGISTRY ?= 
IMAGE_PREFIX ?= mdnwebdocs
else
REGISTRY ?= quay.io/
IMAGE_PREFIX ?= mozmar
endif
```

Reply:
@safwanrahman I'm not sure I understand your concern yet. For local builds, one could easily do:
```
make build-base IMAGE_PREFIX=mdnwebdocs
```
but for now, we really don't want everyone to use `mdnwebdocs`. Once we move completely to the IT-owned infrastructure, we can make `mdnwebdocs` the default value for `IMAGE_PREFIX`.